src/elexmodel/models/BootstrapElectionModel.py

```diff
@@ -845,10 +865,24 @@ def compute_bootstrap_errors(
         #   (it is estiamted as the average error in OLS over all the units in a contest)
         #   delta is the unit level error that is unaccounted for by the contest level effect
         residuals_y, epsilon_y_hat, delta_y_hat = self._estimate_model_errors(
-            ols_y, x_train, y_train, aggregate_indicator_train
+            model_y,
+            x_train,
+            y_train,
+            weights_train,
+            aggregate_indicator_train,
+            K=5,
```

Comment:
Just curious, how did you pick the default of 5 here (and elsewhere)?  I mean I've done (and seen) 3-fold, 5-fold, 10-fold CV but usually my choice there depends on the number of examples in the data set and how much computing power I have 🤔